widget/textgrid.go

```diff
@@ -175,6 +186,7 @@ func (t *TextGrid) RowText(row int) string {
 
 // SetRow updates the specified row of the grid's contents using the specified content and style and then refreshes.
 // If the row is beyond the end of the current buffer it will be expanded.
+// Tab characters are not padded with spaces.
```

Comment:
I'd consider rewording this slightly so it does not sound like a warning.
Maybe `The cells content will be set directly and no interpretation of tabs or spaces will occur`?

Reply:
You want `RowText()` to support tabs, shouldnt `SetRow()` also support tabs?

Reply:
Hmm, well `SetRow` is not `SetRowText` the text helpers do tab conversion, the code that accesses cells directly does not

Reply:
Ah my bad, didnt read the method. Odd that you don't have a SetRowText